templates/angular/igx-ts/grid/default/files/src/app/__path__/__name__.component.ts

```diff
@@ -14,4 +15,10 @@ export class $(ClassName)Component implements OnInit {
   ngOnInit() {
     this.localData = employeesData;
   }
+
+  public onColumnInit(column: IgxColumnComponent) {
+    if (column.field === 'RegistererDate') {
+      column.formatter = (date => date.toDateString());
```

Comment:
I'd use `toLocaleDateString` here - it's localized and usually shorter format that fits better in this particular grid :)

Reply:
Done!